From cc9b7cd3949719f50f570b8e79e82c1e0a369d67 Mon Sep 17 00:00:00 2001
From: Valentin Caron <valentin.caron@foss.st.com>
Date: Wed, 26 Oct 2022 15:13:43 +0200
Subject: [PATCH] ARM: dts: stm32: add power-domains in stm32mp15x uart nodes

On stm32mp15 soc, add UART devices to "pd_core" power domain.

Signed-off-by: Erwan Le Ray <erwan.leray@foss.st.com>
Signed-off-by: Valentin Caron <valentin.caron@foss.st.com>
Change-Id: Ie1b5cf90285d4c348e9362c53a0986f64a1a08a3
---
 arch/arm/boot/dts/st/stm32mp151.dtsi | 7 +++++++
 1 file changed, 7 insertions(+)

--- a/arch/arm/boot/dts/st/stm32mp151.dtsi
+++ b/arch/arm/boot/dts/st/stm32mp151.dtsi
@@ -495,6 +495,7 @@
 			interrupts-extended = <&exti 27 IRQ_TYPE_LEVEL_HIGH>;
 			clocks = <&rcc USART2_K>;
 			wakeup-source;
+			power-domains = <&pd_core>;
 			dmas = <&dmamux1 43 0x400 0x15>,
 			       <&dmamux1 44 0x400 0x11>;
 			dma-names = "rx", "tx";
@@ -507,6 +508,7 @@
 			interrupts-extended = <&exti 28 IRQ_TYPE_LEVEL_HIGH>;
 			clocks = <&rcc USART3_K>;
 			wakeup-source;
+			power-domains = <&pd_core>;
 			dmas = <&dmamux1 45 0x400 0x15>,
 			       <&dmamux1 46 0x400 0x11>;
 			dma-names = "rx", "tx";
@@ -532,6 +534,7 @@
 			interrupts-extended = <&exti 31 IRQ_TYPE_LEVEL_HIGH>;
 			clocks = <&rcc UART5_K>;
 			wakeup-source;
+			power-domains = <&pd_core>;
 			dmas = <&dmamux1 65 0x400 0x15>,
 			       <&dmamux1 66 0x400 0x11>;
 			dma-names = "rx", "tx";
@@ -641,6 +644,7 @@
 			interrupts-extended = <&exti 32 IRQ_TYPE_LEVEL_HIGH>;
 			clocks = <&rcc UART7_K>;
 			wakeup-source;
+			power-domains = <&pd_core>;
 			dmas = <&dmamux1 79 0x400 0x15>,
 			       <&dmamux1 80 0x400 0x11>;
 			dma-names = "rx", "tx";
@@ -653,6 +657,7 @@
 			interrupts-extended = <&exti 33 IRQ_TYPE_LEVEL_HIGH>;
 			clocks = <&rcc UART8_K>;
 			wakeup-source;
+			power-domains = <&pd_core>;
 			dmas = <&dmamux1 81 0x400 0x15>,
 			       <&dmamux1 82 0x400 0x11>;
 			dma-names = "rx", "tx";
@@ -747,6 +752,7 @@
 			interrupts-extended = <&exti 29 IRQ_TYPE_LEVEL_HIGH>;
 			clocks = <&rcc USART6_K>;
 			wakeup-source;
+			power-domains = <&pd_core>;
 			dmas = <&dmamux1 71 0x400 0x15>,
 			       <&dmamux1 72 0x400 0x11>;
 			dma-names = "rx", "tx";
@@ -1681,6 +1687,7 @@
 			interrupts-extended = <&exti 26 IRQ_TYPE_LEVEL_HIGH>;
 			clocks = <&rcc USART1_K>;
 			wakeup-source;
+			power-domains = <&pd_core>;
 			status = "disabled";
 		};
 
